pkgs/development/libraries/aws-c-common/default.nix

```diff
@@ -13,6 +13,10 @@ stdenv.mkDerivation rec {
 
   nativeBuildInputs = [ cmake ];
 
+  postPatch = stdenv.lib.optionalString stdenv.hostPlatform.isMusl ''
+    sed -i "24ireturn()" cmake/AwsTestHarness.cmake
```

Comment:
This is fragile. In general, a better approach is to generate a patch with `diff -U1`.

This breaks non-musl configure with:
```
CMake Error at tests/CMakeLists.txt:260 (target_compile_options):
  Cannot specify compile options for target "aws-c-common-tests" which is not
  built by this project.
```
Why does it work with musl?

You could try `-DBUILD_TESTING=OFF` instead.

Reply:
Yeah it’s super broken. BUILD_TESTING doesn’t work at least

Reply:
I see, `BUILD_TESTING` is not actually implemented. You could patch out `add_subdirectory(tests)` in the main `CMakeLists.txt` then.